tests/core/test_expectation_suite.py

```diff
@@ -250,6 +251,7 @@ def test_expectation_suite_equality(baseline_suite, identical_suite, equivalent_
     assert baseline_suite != equivalent_suite  # ne works properly
 
 
+@pytest.mark.unit
 def test_expectation_suite_equivalence(
     baseline_suite,
     identical_suite,
```

Comment:
I find these parameter names confusing. Can we instead describe the expected differences/similarities?